sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/implementation/serializer/DigitalTwinsStringSerializer.java

```diff
@@ -33,8 +34,8 @@ public void serialize(String s, JsonGenerator jsonGenerator, SerializerProvider
 
     private boolean isValidJson(String jsonInString ) {
         try {
-            mapper.readTree(jsonInString);
-            return true;
+            JsonNode node = mapper.readTree(jsonInString);
```

Comment:
Could use some code comments for future maintainers about what is valid, what is invalid, and why